src/app/utility/element-selector.component.html

```diff
@@ -139,8 +139,8 @@
               [inSearchMode]="formData.inSearchMode"
               [doNotMakeSelectedBold]="true"
               [searchCriteria]="formData.treeSearchText"
-              [filterByDomainType]="['Folder', 'CodeSet']"
-              [expandOnNodeClickFor]="['Folder']"
+              [filterByDomainType]="['Folder', 'VersionedFolder', 'CodeSet']"
```

Comment:
I noticed that some of these selections have empty folders as options as things, as an improvement, we could steal the ones I used for the copy screen that try and filter these out. They also do things like only show terminologies for terminology selectors and such

Reply:
At a later date maybe